code-challenges.js

```diff
@@ -8,62 +8,160 @@
+// -return string2 if more characters are present than string1 (from conditional statement else)
+
 // Set one:
 const fruit1 = "apple"
 const fruit2 = "banana"
 // Expected outcome: "banana"
 
+const mostFruitChar = (string1, string2) =>{
+    if (string1.length > string2.length){
+        return string1
+    }
+    else {
+        return string2
+    }
+}
+console.log(mostFruitChar(fruit1, fruit2))
+       //outcome: banana
+
 // Set two:
 const fruit3 = "cherry"
 const fruit4 = "kiwi"
 // Expected outcome: "cherry"
 
+console.log(mostFruitChar(fruit3, fruit4))
+       //outcome: cherry
+
+
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Pseudo code:
+// -input is 1 number from a possible 3 variables (temperature1, temperature2, temperature3)
+// -output is if the test variable is below, at, or above boiling point of 212 degrees
+// -process is to create a function named boilingPoint
+// -pass the parameter of temp into the function for comparison to boiling point. **** learned by accident parameter does not need parentheses (at least in this instance)
+// -determine if variable is below, at or above boiling point using comparison operator (< or >) and boiling point of 212  degrees and conditionals if/else
+// -return (temperature1) 42 is below boiling point if it is below 212 using string interpolation
+// -return (temperature2) 350 is above boiling point if it is above 212 using string interpolation
+// -return (temperature3) 212 is at boiling point if it is equal to 212 using string interpolation
+
+const boilingPoint = temp => {
+    if (temp < 212){
+        return `${temperature1} is below boiling point`;
+    }
+    else if (temp > 212){
+        return `${temperature2} is above boiling point`;
+    }
+    else  {
+        return `${temperature3} is at boiling point`;
+    }
+}
+
 
 const temperature1 = 42
 // Expected output: "42 is below boiling point"
+console.log(boilingPoint(temperature1))
+   // output: 42 is below boiling point
 
 const temperature2 = 350
 // Expected output: "350 is above boiling point"
-
+console.log(boilingPoint(temperature2))
+// output: 350 is above boiling point
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
+console.log(boilingPoint(temperature3))
+// output: 212 is at boiling point
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
+//-input is 2 arrays 
+//-output return length of both arrays combined using .length built in method
+//-process create a function combinedArrays and set parameters to combine arrays
+//-return the output of the combined arrays a + b
+const combinedArrays = (a,b) => {
+    return a + b
+}
```

Comment:
I see the idea, but pay attention to the prompt. The input is 2 arrays, not the lengths of 2 arrays. Also, it is asking you to combine the two arrays, and then return the final length!